src/data.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable prefer-const */
+/* eslint-disable import/no-mutable-exports */
+let currentUnit = 0; // 0 - Celsius 1 - Fahrenheit
+/* eslint-enable import/no-mutable-exports */
+/* eslint-enable prefer-const */
+
+function parseTemp(temp) {
+  const parsedTemp = currentUnit === 0 ? Number(temp) - 273.1 : 1.8 * (Number(temp) - 273) + 32;
+  return Math.round(parsedTemp * 10) / 10;
+}
+
+function getData(weatherData) {
+  const city = weatherData.name;
+  const { country } = weatherData.sys;
+  const place = `${city}, ${country}`;
+  const { main: weatherTitle, description: weatherDesc } = weatherData.weather[0];
+  const details = weatherData.main;
+  const {
+    feels_like: feeling,
+    humidity,
+    pressure,
+    temp,
+    temp_max: maxTemp,
+    temp_min: minTemp,
+  } = details;
+
+  return {
+    place,
+    weatherTitle,
+    weatherDesc,
+    feeling: parseTemp(feeling),
+    humidity,
+    pressure,
+    temp: parseTemp(temp),
+    maxTemp: parseTemp(maxTemp),
+    minTemp: parseTemp(minTemp),
+  };
+}
+
+function inputResult({
+  place,
+  weatherTitle,
+  weatherDesc,
+  feeling,
+  humidity,
+  pressure,
+  temp,
+  maxTemp,
+  minTemp,
+}) {
+  const results = [
+    { elementId: '#result-temp', value: `${temp}°` },
+    { elementId: '#result-place', value: place },
+    { elementId: '#result-weather', value: weatherTitle },
+    { elementId: '#result-weather-desc', value: weatherDesc },
+    { elementId: '#result-feeling', value: `${feeling}°` },
+    { elementId: '#result-humidity', value: `${humidity}%` },
+    { elementId: '#result-pressure', value: `${pressure}mb` },
+    { elementId: '#result-mintemp', value: `${minTemp}°` },
+    { elementId: '#result-maxtemp', value: `${maxTemp}°` },
+  ];
+  /* eslint-disable no-restricted-syntax */
+  for (const { elementId, value } of results) {
+    document.querySelector(elementId).innerText = value;
+  }
+  /* eslint-enable no-restricted-syntax */
```

Comment:
Again, please avoid linter deactivation. Try using a regular loop, or some other way to avoid deactivating linters.